src/components/Menus/Sidebar/Sidebar.js

```diff
@@ -1,10 +1,17 @@
 import React, { useState } from "react";
 import { useTranslation } from "react-i18next";
 
-import { HomeIcon, FileIcon, ImageIcon, InfoIcon } from "../../../Icons";
 import FilePanel from "./FilePanel/FilePanel";
 import InfoPanel from "./InfoPanel/InfoPanel";
 import SidebarBar from "./SidebarBar";
+import SettingsPanel from "./SettingsPanel/SettingsPanel";
+import {
+  HomeIcon,
+  FileIcon,
+  ImageIcon,
+  InfoIcon,
+  SettingsIcon,
+} from "../../../Icons";
 import ProjectsPanel from "./ProjectsPanel/ProjectsPanel";
 
 import "./Sidebar.scss";
```

Comment:
🔧 it only came up because of the settings icon but `sidebar__bar-option` needs to have a width of 44px too